### DISCUSSION TITLE
verite lib as a standalone package

### DISCUSSION THREAD

**Discussion by cwarny** [centrehq/verite#384] [Q&A] [answered]
Hi there, thanks so much for putting this library out, hopefully I can contribute to it at some point. I'm trying to use functionality from the core verite lib into my own UI, and I'm not sure how to build just that part into my own project. I'm not super familiar with workspaces--how would I go about installing just the core lib (`verite/packages/verite`) as a node module? I tried a simple `npm i verite` at the root of my project but that was unsuccessful. I suspect it is due to the workspace setup. Hopefully my question makes sense. Thanks again for the great lib

**Comment by cwarny** [comment, marked as answer]
Well, I figured out the problem. Verite has some dependencies written to work in NodeJS and not necessarily in the browser, so I just had to polyfill the Node-specific feature in my app's build process. Once I did that `npm i verite` seems to be working.

**Comment by kimdhamilton** [comment]
Thanks @cwarny, glad you figured it out. 

One other thing  -- you also may need to run `npm run setup` to set up some other prereqs. Thanks for checking it out!
